include/common_robotics_utilities/math.hpp

```diff
@@ -10,6 +10,7 @@
 #include <vector>
 
 #include <Eigen/Geometry>
+#include <common_robotics_utilities/utility.hpp>
```

Comment:
Now, I don't know the style rules here, but it seems like there should be a blank line between third-party and first-party headers?

```suggestion

#include <common_robotics_utilities/utility.hpp>
```

Reply:
Style elsewhere in CRU has been no newline between Eigen includes and ROS-package includes (including CRU's own headers).